src/lib/ThumbnailsSidebar.js

```diff
@@ -361,6 +368,56 @@ class ThumbnailsSidebar {
             }
         });
     }
+
+    /**
+     * Toggles the thumbnails sidebar
+     * @return {void}
+     */
+    toggle() {
+        if (!this.anchorEl) {
```

Comment:
Since you're already checking for `anchorEl` in each function below you could remove this , if you want